### PR TITLE
Spanish (Spain) translation

### PR DESCRIPTION
**Pull request by treborin**
Supersedes #357

### CODE REVIEW

**Comment by treborin** [comment]
Hi @datamweb ,
Same issues with this second PR.
Please, let me know what can i do. I'll be here to upload files again whenever you find a solution.
Thanks for your help

**Comment by datamweb** [comment]
Hi @treborin,
I'm sorry you took the trouble to participate.
I will discuss this with my colleagues, it may take some time.
In any case we will tag you to know about the condition.

File: src/Language/es/Auth.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<?php
```
Comment (kenjis) [inline src/Language/es/Auth.php:1]: Please fix the file encoding. We only accept UTF-8 file.

```consolse
$ file src/Language/es/Auth.php 
src/Language/es/Auth.php: PHP script text, ISO-8859 text
```

**Review by kenjis — APPROVED** [review]

**Review by kenjis — CHANGES_REQUESTED** [review]
Please fix the file encoding.

File: src/Language/es/Auth.php (hunk 1 above)
Comment (datamweb) [reply src/Language/es/Auth.php:1]: @treborin [here ](https://www.kindacode.com/article/how-to-change-file-encoding-in-vs-code/)is a good guide.

Let me know if you need more help

Comment (treborin) [reply src/Language/es/Auth.php]: image

Comment (treborin) [reply src/Language/es/Auth.php]: image

**Comment by kenjis** [comment]
There still is something wrong.

```console
$ php -l src/Language/es/Auth.php 
PHP Fatal error:  Namespace declaration statement has to be the very first statement or after any declare call in the script in src/Language/es/Auth.php on line 3

Fatal error: Namespace declaration statement has to be the very first statement or after any declare call in the script in src/Language/es/Auth.php on line 3

Errors parsing src/Language/es/Auth.php
```

**Comment by kenjis** [comment]
@treborin Please remove the BOM.

```diff
--- a/src/Language/es/Auth.php
+++ b/src/Language/es/Auth.php
@@ -1,4 +1,4 @@
-<U+FEFF><?php
+<?php^M
 
 namespace CodeIgniter\Shield\Language\es;
 
```

**Comment by kenjis** [comment]
And please remove `^M`. That is the line break should be `LF` only, not `CRLF`.

**Comment by kenjis** [comment]
Okay, thanks!

File: src/Language/es/Auth.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    'unableSendEmailToUser' => 'Lo sentimaos, ha habido un problema al enviar el email. No podemos enviar un email a "{0}".',
+    'throttled'             => 'demasiadas peticiones hechas desde esta IP. Puedes intentarlo de nuevo en {0} segundos.',
+
+    'email'           => 'Dirección Email',
+    'username'        => 'Usuario',
+    'password'        => 'Contraseña',
+    'passwordConfirm' => 'Contraseña (de nuevo)',
+    'haveAccount'     => '¿Ya tienes una cuenta?',
+
+    //Botones
```
Comment (datamweb) [inline src/Language/es/Auth.php:27]:
```suggestion
    // Botones
```

**Review by datamweb — CHANGES_REQUESTED** [review]
Please change

File: src/Language/es/Auth.php (hunk 2 above)
Comment (treborin) [reply src/Language/es/Auth.php]: Done!

**Comment by datamweb** [comment]
@treborin,
Everything is fine, just fix [this](https://github.com/codeigniter4/shield/runs/7818458988?check_suite_focus=true). `(\ No newline at end of file)`

**Comment by treborin** [comment]
> @treborin, Everything is fine, just fix [this](https://github.com/codeigniter4/shield/runs/7818458988?check_suite_focus=true). `(\ No newline at end of file)`

@datamweb , done!

**Comment by treborin** [comment]
wow!!!  finally :)

**Review by datamweb — APPROVED** [review]
**Roberto** thank you for contributing with love to make Shield better.
Where there is love, there is pain.
Gracias!!

**Review by kenjis — APPROVED** [review]

**Comment by kenjis** [comment]
@treborin Do you use "pull bot" ?
It seems  "pull bot" deleted your PR branch.

Could you send this PR again?

**Pull request merged**